ios/RNSScreenStackAnimator.mm

```diff
@@ -442,4 +508,14 @@ - (void)animateTransitionWithStackAnimation:(RNSScreenStackAnimation)animation
   [self animateSimplePushWithShadowEnabled:shadowEnabled transitionContext:transitionContext toVC:toVC fromVC:fromVC];
 }
 
++ (UISpringTimingParameters *)defaultSpringTimingParametersApprox
+{
+  // Default curve provider is as defined below, however spring timing defined this way
+  // ignores the requested duration of the animation, effectively impairing our `animationDuration` prop.
+  // Damping of 1.0 seems close enough and we keep `animationDuration` functional.
+  // id<UITimingCurveProvider> timingCurveProvider = [[UISpringTimingParameters alloc] init];
+
+  return [[UISpringTimingParameters alloc] initWithDampingRatio:1.0];
```

Comment:
I think this PR is a continuation of https://github.com/software-mansion/react-native-screens/pull/2413

I left a comment about this config: https://github.com/software-mansion/react-native-screens/pull/2413/files#r1829111470 :)

Reply:
I've answered there ;D